frontend/src/tests/setupTests.js

```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom';
```

Comment:
Could you comment on this too please? I'm not sure what this is for :smile:

Reply:
A Better approach may be to create a page in your wiki about how Testing is done, how to write tests etc. A condensed version of the actual documents. 

I'm not saying do this, but it's an example of something I've done in the past
https://github.com/bcgov/gwells/wiki/Testing

Reply:
I will be adding comments once the tests method are confirmed to be what we will be pursuing, or else I'm deleting it. 🤣 After I'm done with the project charter, I'm working on loving the readme so don't worry. 😄